doc/userguide/rules/header-keywords.rst

```diff
@@ -156,8 +156,15 @@ direction you would like to match::
   dest: if the destination matches with the given geoip.
   src: the source matches with the given geoip.
 
-The keyword only supports IPv4. As it uses the GeoIP API of Maxmind,
-libgeoip must be compiled in.
+The keyword only supports IPv4. As it uses the GeoIP2 API of MaxMind,
+libmaxminddb must be compiled in.   You must download and install the
```

Comment:
Additional spaces should be avoided

Reply:
Oops!  My "over forty typewriter training" is showing.  I was originally taught two spaces after a period.  It's fixed now.